src/Particle/DistanceTable.h

```diff
@@ -96,15 +96,28 @@ class DistanceTable
   ///returns the number of source particles
   inline size_t sources() const { return num_sources_; }
 
-  /** evaluate the full Distance Table
+  /** evaluate the full Distance Table. Legacy API
    * @param P the target particle set
    */
-  virtual void evaluate(ParticleSet& P) = 0;
+  void evaluate(ParticleSet& P) { evaluate(P.getCoordinates()); }
```

Comment:
What is the logic behind the loss of virtual on evaluate()? If it was needed before, it should still be needed. The function changes from pure virtual to virtual.

Reply:
The virtual is shifted to the newly introduced `evaluate` below. The legacy API is no longer virtual.